tests/testautorelay.nim

```diff
@@ -95,21 +95,25 @@ suite "Autorelay":
     relayClient = RelayClient.new()
     proc checkMA(addresses: seq[MultiAddress]) =
       if state == 0 or state == 2:
```

Comment:
Why do we need this `state` variable and what's the reasoning behind the conditions?